src/components/Account.jsx

```diff
@@ -24,10 +23,10 @@ const Account = () => {
     dispatch(incrementByamount())
   }
 
-  useEffect(() => {
-    const timeOut = setTimeout(() => setSignup(true), 3000);
-    return () => timeOut;
-  }, []);
```

Comment:
You need to look for another way to solve this problem. Using setTimeout on a web page is not deterministic.

Reply:
Ok... thanks Sir